package.json

```
@@ -4,8 +4,8 @@
   "description": "Making Memeing Maker: A game for creating and sharing meaningful memes",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "postinstall": "npm run build",
+    "test": "./node_modules/.bin/mocha --exit --compilers js:@babel/register",
+    "post:install": "npm run build",
```

@Avuidrauxs 
Is there a reason for this change? 
```suggestion
    "postinstall": "npm run build",
```

---

I did this so the build version can be created the immediate moment the the modules are installed in production.

---

Yes but `CircleCi` runs it and gets an error with babel parser, so I'm thinking we should rather chain the script like this 
```
npm run post:install && node node/server
```

---

Okay.
So we add it to the start script?

---

Yes that's what I have done now.

---

We can find out why it's failing on CircleCI although the easiest thing to do would be removing the `postinstall` script :)